server/db/models/index.js

```diff
@@ -2,12 +2,15 @@
 import { readdirSync } from 'fs';
 import { basename as _basename, join } from 'path';
 import Sequelize from 'sequelize';
+import dotenv from 'dotenv';
```

Comment:
Why do you need dotenv here?

Reply:
I felt we called process in that file without including the dotenv, and i don't  know if it would build without it even if it was added through another file

Reply:
This file was generated automatically by sequelize-cli. And we're not exactly using the `.env` in this file. The config happens in the `config.js` file.

Reply:
Ok, noted

Reply:
it was written in PT board. https://www.pivotaltracker.com/story/show/166265849